addons/website_event_track/static/src/js/event_track_timer.js

```diff
@@ -0,0 +1,87 @@
+odoo.define('website_event_track.website_event_track_timer', function (require) {
+
+'use strict';
+
+const publicWidget = require('web.public.widget');
+
+/*
+ * Simple implementation of a timer widget that uses a "time to live" configuration
+ * value to countdown seconds on a target element.
+ * Will be used to visually countdown the time before a talk starts.
+ * When the timer reaches 0, the element destroys itself.
+ */
+publicWidget.registry.websiteEventTrackTimer = publicWidget.Widget.extend({
+
+    selector: '.o_we_track_timer',
+    events: {
+        'click .close': '_onCloseClick',
+    },
+
+    /**
+     * @override
+     */
+    start: function () {
+        return this._super.apply(this, arguments).then(() => {
+            let timeToLive = this.$el.data('time-to-leave');
+            let deadline = moment().add(timeToLive, 'seconds');
+            let remainingMs = deadline.diff(moment());
+            if (remainingMs > 0) {
+                this._updateTimerDisplay(remainingMs);
```

Comment:
If you call "refreshTimer()" here instead.
You can embed the code of "_updateTimerDisplay" within "_refreshTimer" directly.
Saving a bit of method calls / complexity.

What do you think?